src/components/ContactsList.js

```diff
@@ -23,7 +23,7 @@ function ContactsList(props) {
         <h2>Contacts</h2>
       </header>
       <ul className="contacts-list">
-        {contacts.map((contact, index) => {
+        {!loading && contacts.map((contact, index) => {
```

Comment:
Nice conditional render here of the loading animation or the rendered address details.